### PR TITLE
*: format manifests with cargo-manifmt

### PR DESCRIPTION
**Pull request by benesch**
Keeping manifests consistenly sorted reduces future diff noise, and
reduces confusion about equivalent constructs, like "serde = 1" vs
"serde = 1.0.0".

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5164)

### CODE REVIEW

File: src/avro-derive/Cargo.toml [hunk 1]
```diff
@@ -5,12 +5,10 @@ authors = ["Brennan Vincent <brennan@umanwizard.com>"]
 license = "Apache-2.0"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (quodlibetor) [inline src/avro-derive/Cargo.toml:8]: did you do this or does cargo-maniformat strip comments?

Comment (quodlibetor) [reply src/avro-derive/Cargo.toml:8]: oh nvm I see a comment is preserved below.

Comment (benesch) [reply src/avro-derive/Cargo.toml:8]: It's... complicated: https://github.com/benesch/cargo-manifmt#limitations

In this case though getting rid of this comment seems correct. No point in having this clutter around.

Comment (quodlibetor) [reply src/avro-derive/Cargo.toml:8]: Yeah I agree with deleting that one! Seems fine for something you're going to run occasionally. Thanks!

**Review by quodlibetor — APPROVED** [review]

**Comment by benesch** [comment]
Thanks for the holiday review!

**Pull request merged**